test/specs/modules/Dimmer/Dimmer-test.js

```diff
@@ -29,13 +29,13 @@ describe('Dimmer', () => {
   })
 
   describe('active', () => {
-    it('removes the `disabled` className when true', () => {
+    it('adds the `transition visible` className when true', () => {
```

Comment:
The common test can be configured to assert a className that is different than the prop name.  Should be able to replace this describe block with:

```jsx
common.propKeyOnlyToClassName(Dimmer, 'active', {
  className: 'active transition visible',
})
```